app/views/shared/_postcode-lookup.html.erb

```diff
@@ -2,7 +2,7 @@
   input_label ||= t("coronavirus_local_restrictions.lookup.input_label")
   input_value ||= nil
   input_error ||= nil
-  hint_text ||= sanitize(t("coronavirus_local_restrictions.lookup.hint_text"))
+  hint_text ||= render_govspeak(t("coronavirus_local_restrictions.lookup.hint_text"))
```

Comment:
Could you either squash the two commits together, or re-arrange them so that this change, and the change to locale file are in the right place? This commit should only have the test changes in it.